baseplate/retry.py

```diff
@@ -0,0 +1,126 @@
+"""Policies for retrying an operation safely."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import time
+
+
+class RetryPolicy(object):
+    """A policy for retrying operations.
+
+    Policies are meant to be used as an iterable::
```

Comment:
Yeah, it's a reStructuredText shorthand that starts a block after the current paragraph.